src/coreclr/nativeaot/System.Private.CoreLib/src/System/Exception.CoreRT.cs

```diff
@@ -166,14 +166,14 @@ private static void AppendExceptionStackFrame(object exceptionObj, IntPtr IP, in
 
         internal DispatchState CaptureDispatchState()
         {
-            IntPtr[] stackTrace = _corDbgStackTrace;
+            IntPtr[]? stackTrace = _corDbgStackTrace;
             if (stackTrace != null)
             {
                 IntPtr[] newStackTrace = new IntPtr[stackTrace.Length];
                 Array.Copy(stackTrace, 0, newStackTrace, 0, stackTrace.Length);
                 stackTrace = newStackTrace;
             }
-            return new DispatchState(stackTrace);
+            return new DispatchState(stackTrace!);
```

Comment:
```suggestion
            return new DispatchState(stackTrace);
```
?

Reply:
Seems to be I made mistake. Will change declaration of  `DispatchState` constructor.